superset-frontend/src/components/Checkbox/index.tsx

```diff
@@ -25,8 +25,8 @@ import {
 
 interface CheckboxProps {
   checked: boolean;
-  onChange: (val?: boolean) => {};
-  style: object;
+  onChange: (val?: boolean) => void;
+  style?: object;
```

Comment:
are these changes intentional? I don't remember them before

Reply:
_If_ it _is_ intentional (I'm not sure either), it'd be the only remaining instance of `object` so maybe we can touch that up too?

Reply:
Shouldn't it be `import { CSSProperties } from 'react'`?